### DISCUSSION TITLE
max_conns inside upstream block

### DISCUSSION THREAD

**Discussion by Samhamsam** [nginx-proxy/nginx-proxy#1710] [Q&A] [answered]
Is it possible to add max_conns for a subdomain to an upstream block with a custom configuration file?

**Comment by tkw1536** [comment, marked as answer]
It's not currently possible to add anything to the `upstream` block without modifying [nginx.tmpl](https://github.com/nginx-proxy/nginx-proxy/blob/main/nginx.tmpl).
